tasks/main.yml

```diff
@@ -65,6 +49,20 @@
     mode: 0440
   notify:
     - restart prometheus-omero-exporter
+  when: ansible_os_family == "RedHat"
+
+
+- name: omero prometheus exporter | configure service
+  become: true
+  template:
+    dest: /etc/prometheus-omero-exporter
```

Comment:
Note that this task could also been combined with the previous ones to use `environment_file_path[ansible_os_family]`. 

Functionally, there should be no functional difference between using this style, `when` in the tasks or even splitting OS-specific tasks into separate subplaybooks e.g. as in https://github.com/ome/ansible-role-postgresql-client. My only recommendation would be to reduce the variations to facilitate your maintenance moving forward